src/osx/webview_webkit.mm

```diff
@@ -503,6 +506,50 @@ function createClass(name,rules) { \
         return false;
 }
 
+bool wxWebViewWebKit::ClearBrowsingData(int types, wxDateTime since)
+{
+    if ( WX_IS_MACOS_AVAILABLE(10, 11) )
+    {
+        NSSet<NSString*>* clearDataTypes = nil;
+        if (types & wxWEBVIEW_BROWSING_DATA_OTHER)
+        {
+            return false;
```

Comment:
Again, it looks like we could map this to `WKWebsiteDataTypeServiceWorkerRegistrations`?